cotainr/container.py

```diff
@@ -147,14 +147,27 @@ def add_metadata(self):
             f.seek(0)
             json.dump(metadata, f)
 
+    def _create_file(self, *, env_file):
+        """
+        Create the Path/file `env_file` in the Singularity container.
+
+        Parameters
+        ----------
+        env_file : :class:`pathlib.PosixPath`
+            For example, Path("sandbox_dir/.singularity.d/env/92-cotainr-env.sh")
+        """
+        self._assert_within_sandbox_context()
+
+        # ensure that the file is created *within* the container to get correct permissions, etc.
+        self.run_command_in_container(cmd=f"touch {env_file}")
```

Comment:
I wonder if we should also check the output of this command for error, i.e. check the return code and content of stdout/stderr.

Reply:
True. What I really meant to ask here is if any error handling at that lower level should be combined with the error handling below (i.e., the `if not env_file.exists():` clause) to make sure the error presented to the user, in case the `touch` command fails, makes sense. I haven't looked at the code, so I am not sure if it makes sense or not.

Reply:
The `if not env_file.exists():` is more like a guarantee that the function will always produce the expected state, not to provide an actionable error to the user. So I think it is reasonable to keep the lower level error handling separate, which should always provide an error before reaching `if not env_file.exists():`.